examples/transactions_examples.py

```diff
@@ -281,3 +281,22 @@ async def run(self, ctx: Config):
                 f"transaction proposer: {transaction.proposal_key.address.hex()}"
             )
             self.log.info(f"transaction script: {transaction.script.decode('utf-8')}")
+
+
+class DebuggingFetchTransactionByIdExample(Example):
```

Comment:
Added example which makes a mainnet call to the failing tx ID `ef6e6dcba87c5f853e0ec940dd382d609ffe11109c28ed239843d71618fa1d71`, will remove this before merging changes